docs/en/reference/configuration.rst

```diff
@@ -322,10 +322,10 @@ Now update your ``cli-config.php`` in the root of your project to look like the
     $paths = [__DIR__.'/lib/MyProject/Entities'];
     $isDevMode = true;
 
-    $config = Setup::createAnnotationMetadataConfiguration($paths, $isDevMode);
+    $ORMconfig = Setup::createAnnotationMetadataConfiguration($paths, $isDevMode);
     $entityManager = EntityManager::create(['driver' => 'pdo_sqlite', 'memory' => true], $config);
 
-    return DependencyFactory::fromEntityManager($config, new ExistingEntityManager($entityManager));
+    return DependencyFactory::fromEntityManager($ORMconfig, new ExistingEntityManager($entityManager));
```

Comment:
this is wrong. This one is meant to be `$config` holding the doctrine/migrations config. the ORM Config is meant to be used in the `EntityManager::create` call on the previous line.

Reply:
I guess you are looing at an old commit? I did a second commit that fixed it, my bad.

Reply:
well, Github shows me that your second commit has been done *after* my review